operator/controllers/lifecycle/keptnappversion/controller_test.go

```diff
@@ -185,7 +186,13 @@ func setupReconciler() (*KeptnAppVersionReconciler, chan string, *fake.ITracerMo
 		UnbindSpanFunc: func(reconcileObject client.Object, phase string) error { return nil },
```

Comment:
it seems like the tests for this component were quite outdated, as they do not contain a test case for an appVersion that actually contains any WorkloadInstances - can you please also add a test case for this to also cover the introduced changes?